.github/workflows/sub-ci-integration-tests-gcp.yml

```
@@ -422,7 +419,7 @@ jobs:
       app_name: zebrad
       test_id: rpc-get-block-template
       test_description: Test getblocktemplate RPC method via Zebra's rpc server
-      test_variables: "NETWORK=${{ inputs.network || vars.ZCASH_NETWORK }},NEXTEST_PROFILE=rpc-get-block-template,ZEBRA_CACHE_DIR=/home/zebra/.cache/zebra"
+      test_variables: "ZEBRA_NETWORK__NETWORK=${{ inputs.network || vars.ZCASH_NETWORK }},NEXTEST_PROFILE=rpc-get-block-template"
```

The `needs.sync-full-mainnet.result == 'success'` condition implies this will only run on Mainnet, the test itself hard codes Mainnet but should work fine on Testnet too if it has an up-to-date Testnet cache.

```suggestion
      test_variables: "NEXTEST_PROFILE=rpc-get-block-template"
```

Update: Not a blocker.